lib/OpenQA/Worker/Commands.pm

```diff
@@ -184,7 +184,7 @@ sub websocket_commands {
             }
         }
         else {
-            log_error("got unknown command $type");
+            log_error("got unknown command '$type'. Are the worker and openQA running compatible versions?");
```

Comment:
For this it would be good to know which worker, no?

Reply:
i would rather opt for a real solution instead of hints - e.g. version check and proper 'die' in case there is no protocol compatibility

Reply:
@coolo, this message is displayed at the output of /usr/share/openqa/scripts/worker --verbose --instance X

So you know which worker it is.

----

@mudler, at least this change is better than before, now the users know the reason why, and how to solve the problem, since this version incompatibility is not (yet?) documented.

Reply:
@mudler *you* rejected the ticket which means to me: "We don't want to work on it". And now you are saying we should implement more? I'm confused

Reply:
No, he rejected the ticket saying the solution in https://progress.opensuse.org/issues/7086 is preferred. Not sure which 'we' your 'we's refer to above, but 7086 is part of the backlog. And I disagree with code asking questions in error logs

Reply:
@okurz, feel free to ping me on IRC or comment on the issue before getting an unclear picture - ticket rejection doesn't means that to me, it means invalid. 

To add  to what  coolo already said, imho as posed, the issue is regarding the message appearing in the journal as a bug - but that's not a bug - since we want to be aware if the worker cannot execute an action depending on the message received, and since doesn't always means it's a version incompatibility between protocols i think it might be misleading too.
That said, the resolution to that ticket, is having workers and webui at the same version.

My bad if I was unclear - but as @coolo referenced the related issue in that poo (very fast!) - i thought it was unnecessary to add even more.

By the way, doesn't seems to me I asked you to implement anything ;)